### ISSUE TITLE
Default account

### ISSUE THREAD

**Issue by lstarlitsky** [gamegos/cesi#56]
I can't login with admin:admin while first run it, it raise an exception:
```
(sqlite3.IntegrityError) UNIQUE constraint failed: users.username [SQL: 'INSERT INTO users (username, password, usertype) VALUES (?, ?, ?)'] [parameters: ('admin', 'pbkdf2:sha256:50000$bO2jbFQ4$6df2181d675615bd2fc4fcb9e61fd24a77765317067a1af028447f8ea4d673a0', 0)] (Background on this error at: http://sqlalche.me/e/gkpj)
```
however, it works until I am deleting the record of `username=admin` in users table.

**Comment by lstarlitsky** [comment]
the activity.log shows that  default account may be changed in lastest commit
```
$ tail -f activity.log 
2018-11-09 16:47:18,099 [INFO]: admin logged in.
2018-11-09 16:47:29,422 [INFO]: 'emre' user added by 'admin' user.
2018-11-09 16:47:36,632 [ERROR]: admin user changed the own password.
2018-11-09 16:47:59,105 [INFO]: admin logged in.
```

**Comment by f9n** [comment]
Sorry. When I released `Cesi-2.6.2`, I didn't delete my old `users.db` and `activity.log` files.
Now, I'll add these files to the `.gitignore` file and publish the new version.

**Comment by f9n** [comment]
> Sorry. When I released `Cesi-2.6.2`, I didn't delete my old `users.db` and `activity.log` files.
> Now, I'll publish the new version.

https://github.com/gamegos/cesi/releases/tag/v2.6.3
